dispatch/static/manager/src/js/components/Editor/tabs/DeliveryTab.js

```diff
@@ -76,6 +81,12 @@ export default function DeliveryTab(props) {
         {warningMessage}
       </FormInput>
 
+      <FormInput label='Schedule Notification'>
+        <DateTimeInput
+          value={props.scheduled_notification}
+          onChange={dt => updateScheduledNotification(props.update, dt)} />
```

Comment:
I would just do this:

```js
dt => props.update('scheduled_notification', dateObjToAPIString(dt))
```

Reply:
Also, can you add a way to remove (cancel) a notification once it has been created?